src/VisualStudio/CSharp/Impl/CSharpPackage.cs

```diff
@@ -155,5 +156,13 @@ protected override string RoslynLanguageName
                 return LanguageNames.CSharp;
             }
         }
+
+        public override AbstractObjectBrowserLibraryManager LibraryManager
```

Comment:
AbstractObjectBrowserLibraryManager has a ton of methods that really shouldn't be called outside of it. Also, exposing this here causes other types in the ObjectBrowser namespace to leak out as well. Would it be better to have a GetSymbolNavInfo() that is overriden by CSharpPackage and VisualBasicPackage?